src/java.base/windows/native/libjava/ProcessImpl_md.c

```diff
@@ -343,6 +349,16 @@ static jlong processCreate(
                     {
                         win32Error(env, L"CreateProcess");
                     } else {
+                        /* SapMachine 2024-07-01: process group extension */
+                        if (processGroup) {
+                            __declspec(align(8)) HANDLE hJob = CreateJobObject(NULL, NULL);
+                            if (!hJob) {
+                                win32Error(env, L"CreateJobObject");
+                            }
+                            (*env)->SetLongArrayRegion(env, processGroup, 0, 1, (jlong *) &hJob);
```

Comment:
win32Error will throw an exception, so you should not call SetLongArrayRegion with a pending exception. And in that case we should not call AssignProcessToJobObject.

Reply:
ok, I guess youre right. Fixed.